remoter-client/src/wrapper.r

```diff
@@ -0,0 +1,100 @@
+require("RJSONIO")
+require("paws")
+require("zeallot")
+
+reload_from_s3 <- function(experiment_id) {
+    s3 <- paws::s3()
+    bucket_name <- paste("biomage-source", "staging", sep = "-")
+
+    c(body, ...rest) %<-% s3$get_object(
+        Bucket = bucket_name,
+        Key = paste(experiment_id, "r.rds", sep = "/")
+    )
+
+    obj <- readRDS(rawConnection(body))
+    return(obj)
+}
+
+
+run_step <- function(task_name, scdata, config) {
+    switch(task_name,
+        test_fn = {
+            import::from("test_fn.r", task)
+        },
+        stop(paste("Invalid task name given:", task_name))
```

Comment:
why do we call the test function in here?

Reply:
Correct. `test_fn` is just a dummy function that we can use to test the thing working properly. As bioinformaticians submit more code, they will have to add their tasks to this switch.